packages/mermaid/src/diagrams/context-map/parser/contextMap.jison

```diff
@@ -0,0 +1,74 @@
+
+%{
+  function edgeType(text){
+    return text.includes(",") ? text.split(",") : [text]
+  }  
+%}
+
+%lex
+
+%x comment
+%s context
+
+%%
+
+\s+                                { /* skip */ }
+"/*"                               { this.pushState("comment"); }
+<comment>"*/"                      { this.popState(); }
+<comment>[^"*"]|[^"/"]             { /* skip */ }
```

Comment:
If I understood this right
1. `[]` is a symbol class so you do not need to wrap values in quotes here, by adding `"` you add the symbol double quote to the symbol class as well. If you can escape them with `\` if needed. So your class excludes `"` (twice) and `*` from grammar, and second one excludes `"` (twice) and `/`, it should be
  ```
  ^[\*\/]
  ```
2. Does it mean that we prohibit * and / from comments? What if I would like to make such a comment /*******/ ?